fpga.chart.py

```diff
@@ -177,12 +191,20 @@ def _get_fpga_power(self, idx):
             return self._get_xilinx_fpga_power(idx)
 
 
+    def get_fpga_os_status(self):
+        while True:
+            time.sleep(self.temp_power_update_interval)
+            for idx in range(self.fpga_count):
```

Comment:
This adds to the time it takes. Either spawn a thread per FPGA or make the collection such, that it is asynchronous (I prefer the latter).